app/controllers/application_controller.rb

```diff
@@ -4,6 +4,9 @@
 
 class ApplicationController < ActionController::Base
   include Clearance::Controller
+  include Pundit::Authorization
+  after_action :verify_authorized unless :health_check
```

Comment:
This isn't working how you think it might be.  The symbol `:health_check` evaluates to true, so it will never apply the `after_action`.

The proper syntax would be `after_action :verify_authorized, except: %i[health_check]`

However, you'll find that the `index` route will raise an exception since `authorize` is not being called.  The pundit README provides a good example to enforce `policy_scope` on `index` and `authorize` on all other actions. Combined with an exception for `health_check` it would look like this:

```
after_action :verify_pundit_authorization, except: %i[health_check]

...

private

def verify_pundit_authorization
  if action_name == "index"
    verify_policy_scoped
  else
    verify_authorized
  end
end
```